components/os/component.go

```diff
@@ -123,7 +123,8 @@ func (c *component) Check() components.CheckResult {
 	if err != nil {
 		cr.err = err
 		cr.health = apiv1.HealthStateTypeUnhealthy
-		cr.reason = fmt.Sprintf("error getting uptime: %s", err)
+		cr.reason = "error getting uptime"
```

Comment:
We also need to consider deleting the component.  What's the `VirtualizationEnvironment`  used for?

Reply:
Yeah let us remove once machine info is working well :)

`VirtualizationEnvironment` is not really being used in the os component itself. Here just providing as an info.

Reply:
I mean that we would add it in the machine info if we need it.

Reply:
Yes, let me know when we need it, we can add this to the machine info.